### PR TITLE
grpc: fix nil arguments

### PR DESCRIPTION
**Pull request by josedonizetti**
Fixes https://github.com/aquasecurity/tracee/issues/3821

### CODE REVIEW

File: pkg/server/grpc/event_data.go [hunk 1]
```diff
@@ -77,6 +73,10 @@ func getEventValue(arg trace.Argument) (*pb.EventValue, error) {
 // based on the value type
 func parseArgument(arg trace.Argument) (*pb.EventValue, error) {
 	switch v := arg.Value.(type) {
+	case nil:
```
Comment (geyslan) [inline pkg/server/grpc/event_data.go:76]: Should we consider a default case as well?

Comment (josedonizetti) [reply pkg/server/grpc/event_data.go:76]: what is the default case?

Comment (josedonizetti) [reply pkg/server/grpc/event_data.go:76]: We don't return error if the type is not supported, we try to convert it to a structure, and log a message if we can't. We want to send the event even if an argument isn't supported yet: 

https://github.com/aquasecurity/tracee/blob/main/pkg/server/grpc/event_data.go#L341
https://github.com/aquasecurity/tracee/blob/main/pkg/server/grpc/event_data.go#L731

Comment (geyslan) [reply pkg/server/grpc/event_data.go:76]: Got it! 👍🏻

File: pkg/server/grpc/event_data.go [hunk 2]
```diff
@@ -42,6 +42,11 @@ func getEventData(e trace.Event) ([]*pb.EventValue, error) {
 			return nil, err
 		}
 
+		// if convertStruct was not able to convert an argument is because we don't support the convertion
+		if eventValue == nil {
+			continue
```
[the remark this answers is not in the document]
Comment (josedonizetti) [reply pkg/server/grpc/event_data.go:47]: It should never happen to our own types, yeah, but user provided types on signature data, might happen if `convertStruct` can't conver the data: 

https://github.com/aquasecurity/tracee/blob/main/pkg/server/grpc/event_data.go#L740
https://github.com/aquasecurity/tracee/blob/main/pkg/server/grpc/event_data.go#L757

we log the convertion to structure which is our default case https://github.com/aquasecurity/tracee/blob/main/pkg/server/grpc/event_data.go#L732, but I guess a log here also worth.

**Review by yanivagman — APPROVED** [review]
LGTM

**Pull request merged**